_overviews/FAQ/index.md

```diff
@@ -108,20 +108,25 @@ Your sbt 1.x build definition is always a Scala 2.12 program.
 Regardless, in your `build.sbt`, you can set `scalaVersion` to whichever
 available distribution you want and your program code will be compiled with that version.
 
-### I want Scala 3.1.1 (etc); why does std lib say it's using Scala 2.13?
+### I want Scala 3. Why does `versionNumberString` say I'm on 2.13?
 
-Scala 3 currently uses the Scala 2.13 library by leveraging its seamless
-interoperability. Note that it does not necessarily ingest the latest
-version of the Scala 2.13 library.
+To aid migration, Scala 3 currently uses the Scala 2.13 library as-is.
```

Comment:
According to all the crossword puzzle clues, "AS IS" is the answer to, "Common classified clause" or "Flea-market condition", i.e., "Won't fix."

"...library, supplemented by a few bits that are specific to Scala 3."